spec/wowrb_spec.rb

```diff
@@ -1,11 +1,46 @@
 require 'spec_helper'
+      
+  describe Wowrb do
 
-describe Wowrb do
-  it 'has a version number' do
-    expect(Wowrb::VERSION).not_to be nil
+    it 'has a version number' do
+      expect(Wowrb::VERSION).not_to be_nil
+    end
+  
+    it 'has a key value' do 
+      expect(Wowrb.config.api_key).not_to be_empty
+      expect(Wowrb.config.api_key).not_to be_nil
+    end
+  
+    it 'has a region value' do 
+      expect(Wowrb.config.region).not_to be_empty
+      expect(Wowrb.config.region).not_to be_nil
+    end
+  
+    it 'has a locale value' do 
+      expect(Wowrb.config.locale).not_to be_empty
+      expect(Wowrb.config.locale).not_to be_nil
+    end 
+  
+    it 'has not a key value' do     
+      expect(Wowrb.region_leaderboard).
+        to eq("Env variables missing: ENV['BATTLE_NET_KEY']")
```

Comment:
Here and further: https://github.com/arbox/ruby-style-guide/blob/master/README-ruRU.md#consistent-multi-line-chains variant A